lib/utilities/monitoringHandler.js

```diff
@@ -3,65 +3,65 @@ const client = require('prom-client');
 
 const collectDefaultMetrics = client.collectDefaultMetrics;
 const numberOfBuckets = new client.Gauge({
-    name: 'cloud_server_number_of_buckets',
+    name: 's3_cloudserver_buckets',
```

Comment:
should have a unit : `***_count`

Reply:
Following the suggestions here https://www.robustperception.io/on-the-naming-of-things/ and https://docs.google.com/document/d/1qw9P3p7VuFba_KbxJahPgUSxzusGSo2OPCSGLA3MTVg, avoiding the suffix because it is a gauge and not counter.

Reply:
That suggestion is "do not use the `count` or `total`" suffix for gauges, fine.

Anyway my point is not about the specific word used, but that semantically it is not clear what a `s3_cloudserver_buckets` metric reprensents : and i think we would benefit from having suffix ("unit" or otherwise) to better convey what this metrics represents...

(though specifically here I would agree to disagree that `count` implies a counter : i think a user of the dashboard would never expect that the "bucket_count" is a (prometheus) counter and can never go down...)

Reply:
The same convention is used in other components, meaning no suffixes such as `count` or `total` for a gauge. So not modifying it.

Reply:
not sure I understand the reasoning here : since we are currently renaming all metrics, now is the time to cleanup these as well... The goal here is to have meaningful and clear names, and the (overall) guidelines is that metrics should have a unit : so it seems there is something to do indeed...

Reply:
does `s3_cloudserver_total_buckets` make sense? just trying not to use it as a suffix so that it is distinguishable from a counter!

Reply:
updated to include `count`